tests/test_fields.py

```diff
@@ -297,3 +300,32 @@ def to_item(self):
 
         assert fields["field2"].name == "field2"
         assert fields["field2"].meta is None
+
+
+def test_field_subclassing():
+    class Page(ItemPage):
+        @field
+        def field1(self):
+            return 1
+
+        @field
+        def field3(self):
+            return 1
+
+    assert list(get_fields_dict(Page)) == ["field1", "field3"]
+    assert get_fields_dict(Page)["field3"].meta is None
+
+    class Page2(Page):
+        @field
+        def field2(self):
+            return 1
+
+        @field(meta={"foo": "bar"})
+        def field3(self):
+            return 1
+
+    assert get_fields_dict(Page2)["field3"].meta == {"foo": "bar"}
+    assert list(get_fields_dict(Page2)) == ["field1", "field3", "field2"]
+
+    assert get_fields_dict(Page)["field3"].meta is None
+    assert list(get_fields_dict(Page)) == ["field1", "field3"]
```

Comment:
Should we also add a case for symmetry, where `Page` has a field like:
```python
@field(meta={"a": "b"})
def field4(self):
    return 1
```

But then on the `class Page2(Page)` subclass, it became:

```python
@field
def field4(self):
    return 1
```

Reply:
Done in https://github.com/scrapinghub/web-poet/pull/70/commits/26b9508a450cb975251de59bf7218fa8a3173dd3